test/test.expected

```diff
@@ -7,3 +7,12 @@
   )
  )
 )
+(module
+ (type $i32_i32_=>_i32 (func (param i32 i32) (result i32)))
+ (func $0 (param $0 i32) (param $1 i32) (result i32)
```

Comment:
@ospencer Interesting that when this is read from the binary, the `$adder` name is gone. Is that expected?

Reply:
If the function wasn't exported, I suppose there wouldn't be any reason why it would keep the name, but I'm not 100% sure.